explorer/src/api/index.ts

```diff
@@ -5,47 +5,50 @@ import {
   BLOCK_API,
 } from './constants';
 
+import { MixNodeResponse, GatewayResponse, ValidatorsResponse } from '../typeDefs/node-status-api-client'
+
 export class Api {
-  static fetchMixnodes = async () => {
+
+  static fetchMixnodes = async (): Promise<MixNodeResponse> => {
     try {
       const res = await fetch(MIXNODES_API);
       const json = await res.json();
       return json;
     } catch (error: any) {
       console.log('error ', error);
-      throw error.message;
+      throw error;
     }
   };
 
-  static fetchGateways = async () => {
+  static fetchGateways = async (): Promise<GatewayResponse> => {
     try {
       const res = await fetch(GATEWAYS_API);
       const json = await res.json();
       return json;
-    } catch (error) {
+    } catch (error: any) {
       console.log('error ', error);
       return error;
     }
   };
```

Comment:
```suggestion
static fetchGateways = async (): Promise<GatewayResponse> => {
  const res = await fetch(GATEWAYS_API);
  return await res.json();
};
```

Reply:
A couple of things:
- `error:any` should just be `error` because it is already typed as `Error`
- logging to the console should be done by the caller
- the error should be thrown and not returned because the signature return type is `Promise<GatewayResponse>` and not `Promise<GatewayResponse | Error>` (which is not a good idea because Promise already has a mechanism for throwing errors)